Podfile

```diff
@@ -50,7 +50,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 8.7', '>= 8.7.1'
+  pod 'WordPressKit', '~> 8.8'
```

Comment:
Notice that the addition of `tag` to `PostServiceRemoteOptions` in `8.8.0` was a breaking change and has been reverted via https://github.com/wordpress-mobile/WordPressKit-iOS/pull/639  to unblock the [23.6](https://github.com/wordpress-mobile/WordPress-iOS/milestone/260) code freeze.

See also  https://github.com/wordpress-mobile/WordPressKit-iOS/pull/634/files#r1375696151